overlay/base/usr/lib/system/_os/cli/install.py

```
@@ -237,36 +241,11 @@ def install(  # noqa: PLR0917
-        "/bin/bash",
-        "-c",
-        "grub-mkconfig -o /boot/efi/EFI/grub/grub.cfg",
-    )
-    execute(
-        "chroot",
-        sysPath,
-        "/bin/bash",
-        "-c",
+    update_grub_config(sysroot)
+    chroot(
+        list(deployments(sysroot))[0],
```

Ruff (0.16.0)

[warning] Prefer `next(iter(deployments(sysroot)))` over single element slice

Replace with `next(iter(deployments(sysroot)))`

(RUF015)